rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -1089,17 +1223,25 @@ error_code cellGemPrepareVideoConvert(vm::cptr<CellGemVideoConvertAttribute> vc_
 
 	const auto vc = *vc_attribute;
 
-	if (!vc_attribute || vc.version == 0 || vc.output_format == 0 ||
-		(vc.conversion_flags & CELL_GEM_COMBINE_PREVIOUS_INPUT_FRAME && !vc.buffer_memory))
+	if (!vc_attribute || vc.version != CELL_GEM_VERSION)
 	{
 		return CELL_GEM_ERROR_INVALID_PARAMETER;
 	}
 
-	if (vc.video_data_out & 0x1f || vc.buffer_memory & 0xff)
+	if (vc.output_format != CELL_GEM_NO_VIDEO_OUTPUT)
 	{
-		return CELL_GEM_ERROR_INVALID_ALIGNMENT;
+		if (!vc.video_data_out || ((vc.conversion_flags & CELL_GEM_COMBINE_PREVIOUS_INPUT_FRAME) && !vc.buffer_memory))
+		{
+			return CELL_GEM_ERROR_INVALID_PARAMETER;
+		}
 	}
 
+	// TODO:
+	//if (vc.video_data_out & 0x1f || vc.buffer_memory & 0xff)
```

Comment:
It seems that alignment values were 1 bit off (should be 0xf and 0x7f respectively)

Reply:
Can you provide the code please?

Reply:
It's a vm::ptr<void>

Reply:
.aligned(x)

Reply:
yes. but is it aligned(0xf) and aligned(0xff)

Reply:
`aligned(16)` and `aligned(128)`, these are common alignment values (vector/cache line)